packages/cloud-cognitive/src/components/UserProfileImage/UserProfileImage.stories.js

```diff
@@ -58,25 +58,30 @@ export const Default = Template.bind({});
 Default.args = {
   backgroundColor: 'light-cyan',
   theme: 'light',
+  size: 'xlg',
```

Comment:
If you created a `defaultProps` object, you could cut down on a few lines of code that are used. Something like:

```
const defaultProps = {
  backgroundColor: 'light-cyan',
  theme: 'light',
  size: 'xlg',
};
```

and then spread them in your `Template.args`:

```
WithGroupIcon.args = {
  ...defaultProps,
  kind: 'group',
};
```

Def nitpicky and not a blocker :D